docs/docsite/rst/conf.py

```diff
@@ -228,7 +228,7 @@
 
 autoclass_content = 'both'
 
-intersphinx_mapping = {'python': ('https://docs.python.org/2/', (None, '../python2-2.7.13.inv')),
+intersphinx_mapping = {'python': ('https://docs.python.org/3/', (None, '../python3-3.6.2.inv')),
```

Comment:
I'm not sure if we should do this now. The docs may as well still reference things from py2. Let's omit this.